try {
    hello();
} catch (error) {
    console.log(error);
}

try {
    hi();
} catch {
    console.log("Error !");
}